apps/web-giddh/src/app/shared/on-boarding/on-boarding.component.scss

```diff
@@ -61,7 +61,7 @@ label .required {
 }
 
 .name-error {
-    color: $scarlet;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine